draft-ietf-wimse-s2s-protocol.md

```diff
@@ -389,7 +387,6 @@ The decoded JWT claims of the WPT from the example above are shown here:
   "ath": "CL4wjfpRmNf-bdYIbYLnV9d5rMARGwKYE10wUwzC0jI",
   "aud": "https://workload.example.com/path",
   "exp": 1728658672,
-  "iss": "wimse://example.com/specific-workload",
```

Comment:
do you also want to remove it from the example jwt above (lines ~362-370)? Probably should but I know that starts to pull the thread of if the signature is valid etc.

Reply:
I'm regenerating the examples in a separate PR. Makes it easier to merge, etc.

Reply:
okay, that works too